src/Test/Framework/Property.hs

```diff
@@ -159,7 +159,7 @@ instance Testable Bool where
   property b = result $ nothing { ok = Just b }
 
 instance (Arbitrary a, Show a, Testable b) => Testable (a -> b) where
-  property f = forAll arbitrary f
+  property = forAll arbitrary
```

Comment:
+0

I think we shouldn't eta-reduce student-facing code but this is under-the-hood stuff.